src/cards/fortify.ts

```diff
@@ -112,4 +112,11 @@ function add(unit: Unit.IUnit, _underworld: Underworld, _prediction: boolean, qu
     }
   });
 }
-export default spell;
+
+function CalcMult(quantity: number): number {
+  // Each Quantity cuts incoming damage in half multiplicatively
+  const damageReduction = Math.pow(DAMGAGE_REDUCTION_PROPORTION, quantity);
```

Comment:
Should we balance this to do 2,4,6,8 instead of 2,4,8,16?

Reply:
I don't think so. It only lasts one turn and the expense scaling is super high. And intuitively + based off the description I think it would be multiplicative